packages/wonder-blocks-birthday-picker/src/components/__tests__/birthday-picker.test.tsx

```diff
@@ -534,27 +568,31 @@ describe("BirthdayPicker", () => {
             jest.useFakeTimers();
         });
 
-        it("should find and select an item using the keyboard", () => {
+        it("should find and select an item using the keyboard", async () => {
             // Arrange
+            const ue = userEvent.setup({
+                advanceTimers: jest.advanceTimersByTime,
+                pointerEventsCheck: PointerEventsCheckLevel.Never,
+            });
```

Comment:
This is cool! By reading the docs, I thought that we always needed to use `setup`, but it looks like it is only needed when we need to create an instance with certain settings.

Reply:
Yeah, exactly! The default instance works well for us 99% of the time - it turns out that WB is particularly complex, with the mocked timers and pointer checks.